assets/js/buttons-examples.js

```diff
@@ -502,6 +502,16 @@ $(document).ready(function () {
             },
           ],
         },
+        {
```

Comment:
Place this in the correct alphabetical order under Special

Reply:
Done!